// Reversed by pbd (https://github.com/rsc-dev/pbd)
package Devialet.IMASlave4U;

message AudioMode {
	required uint32 mode = 1 ;
	
	enum Mode {
		MonoMix = 1;
		StereoLeft = 2;
		StereoRight = 3;
	}
}

message Property {
	required string name = 1 ;
	required string value = 2 ;
}

message Input {
	required string name = 1 ;
	repeated string schemes = 2 ;
	repeated .Devialet.IMASlave4U.Property properties = 3 ;
}

service Configuration {
}